frontend/src/metabase/admin/settings/components/SettingsSetupList.jsx

```diff
@@ -101,28 +105,48 @@ export default class SettingsSetupList extends Component {
     }
 
     return (
-      <div className="px2">
-        <h2>{t`Getting set up`}</h2>
-        <p className="mt1">{t`A few things you can do to get the most out of Metabase.`}</p>
-        <LoadingAndErrorWrapper
-          loading={!this.state.tasks}
-          error={this.state.error}
+      <Flex justifyContent="space-between">
+        <div className="px2">
+          <h2>{t`Getting set up`}</h2>
+          <p className="mt1">{t`A few things you can do to get the most out of Metabase.`}</p>
+          <LoadingAndErrorWrapper
+            loading={!this.state.tasks}
+            error={this.state.error}
+          >
+            {() => (
+              <div style={{ maxWidth: 468 }}>
+                {nextTask && (
+                  <TaskSection
+                    name={t`Recommended next step`}
+                    tasks={[nextTask]}
+                  />
+                )}
+                {tasks.map((section, index) => (
+                  <TaskSection {...section} key={index} />
+                ))}
+              </div>
+            )}
+          </LoadingAndErrorWrapper>
+        </div>
+        <div
+          className="border-left border-brand text-brand px4"
+          style={{ height: 172 }}
         >
-          {() => (
-            <div style={{ maxWidth: 468 }}>
-              {nextTask && (
-                <TaskSection
-                  name={t`Recommended next step`}
-                  tasks={[nextTask]}
-                />
-              )}
-              {tasks.map((section, index) => (
-                <TaskSection {...section} key={index} />
-              ))}
-            </div>
-          )}
-        </LoadingAndErrorWrapper>
-      </div>
+          <Icon name="cloud" size={48} style={{ color: "#B9D8F4" }} />
+          <div className="pb3">
+            <Text className="text-brand mb0">{t`Have your server maintained for you.`}</Text>
+            <Text className="text-brand text-bold">{t`Migrate to Metabase Cloud.`}</Text>
+          </div>
+
+          <ExternalLink
+            className="bordered rounded border-brand bg-brand-hover text-white-hover px2 py1 text-bold text-center"
+            href={"https://www.metabase.com/migrate/from/selfhosted"}
+            target="_blank"
+          >
+            {t`Learn more`}
+          </ExternalLink>
+        </div>
+      </Flex>
```

Comment:
Yeah, I think it would make sense at least to do that with the ones that are in the side margin. The other one that's below the updates-check is too different stylistically.

Reply:
It looks like they all share the same "Learn more" button so maybe one file that exports a MarginHostingCTA and a HostingInfoLink.

Reply:
Good call. I went ahead and extracted the margin bit into `metabase/admin/settings/components/widgets/MarginHostingCTA.jsx` and the learn more button into `metabase/admin/settings/components/widgets/HostingInfoLink.jsx`